src/ansys/dpf/post/static_mechanical_simulation.py

```diff
@@ -1137,17 +1158,22 @@ def stress_eqv_von_mises_elemental(
             If the problem is multi-stage, can take a list of lists of sector numbers, ordered
             by stage.
         phase_angle_cyclic:
-             For cyclic problems, phase angle to apply (in degrees).
+            For cyclic problems, phase angle to apply (in degrees).
         external_layer:
-             Select the external layer (last layer of solid elements under the skin)
-             of the mesh for plotting and data extraction. If a list is passed, the external
-             layer is computed over list of elements.
+            Select the external layer (last layer of solid elements under the skin)
+            of the mesh for plotting and data extraction. If a list is passed, the external
+            layer is computed over list of elements.
         skin:
-             Select the skin (creates new 2D elements connecting the external nodes)
-             of the mesh for plotting and data extraction. If a list is passed, the skin
-             is computed over list of elements (not supported for cyclic symmetry). Getting the
-             skin on more than one result (several time freq sets, split data...) is only
-             supported starting with Ansys 2023R2.
+            Select the skin (creates new 2D elements connecting the external nodes)
+            of the mesh for plotting and data extraction. If a list is passed, the skin
+            is computed over list of elements (not supported for cyclic symmetry). Getting the
+            skin on more than one result (several time freq sets, split data...) is only
+            supported starting with Ansys 2023R2.
+        reduce_mesh:
```

Comment:
what about the external layer?
I'm not sure about the name "reduce" any other ideas, @rlagha @FedericoNegri?

Reply:
@cbellot000 you're right, I'll check what it does in combination with `external_layer`. As for the name `reduce_mesh`, it is definitely just a proposal.